README.md

```diff
@@ -1,6 +1,6 @@
 This is ALGO-ETH BRIDGE developed on Algorand using smart contracts developed with [Reach](reach.sh) and solidity. It is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
 
-**Developed by**: [Sunday Akinbowale](https://github.com/asolpshinning), [Jake ChaseBuilt](https://github.com/jchaselubitz), and powered by [Cooperativ.io](https://github.com/cooperativ-labs)
+**Developed by**: [Sunday Akinbowale](https://github.com/asolpshinning), [Jake Chase-Lubitz](https://github.com/jchaselubitz), and powered by [Cooperativ.io](https://github.com/cooperativ-labs)
 
 We will continue to develop this repo after hackathon submission at this [Cooperative's Github Repo](https://github.com/cooperativ-labs/eth-algo-NFT-bridge)
```

Comment:
@asolpshinning you can actually add comments on a per-line basis. Like I am doing here.